pkg/processor/template_utils_test.go

```diff
@@ -0,0 +1,64 @@
+// Copyright © 2024 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package processor
+
+import (
+	"testing"
+
+	"github.com/conduitio/conduit/pkg/record"
+	"github.com/matryer/is"
+)
+
+func Test_TemplateUtils_ExecuteTrue(t *testing.T) {
+	is := is.New(t)
+	condition := "{{ eq .Metadata.key \"val\" }}"
```

Comment:
Backticks are great in cases when you need to construct a string containing quotes.

```suggestion
	condition := `{{ eq .Metadata.key "val" }}`
```